build/__dotNetCoreRunTimeVersions.sh

```diff
@@ -7,5 +7,5 @@ NET_CORE_APP_20='2.0.9'
 NET_CORE_APP_21='2.1.17'
```

Comment:
Seems like even 2.1 version has updates. Since it is LTS version, could you update that too?
https://github.com/dotnet/core/blob/master/release-notes/2.1/releases.json#L133

Reply:
will do